app/controllers/delegate_reports_controller.rb

```diff
@@ -51,6 +51,13 @@ def update
         if @competition.end_date >= DelegateReport::REPORTS_ENABLED_DATE
           CompetitionsMailer.notify_of_delegate_report_submission(@competition).deliver_later
           CompetitionsMailer.wrc_delegate_report_followup(@competition).deliver_later
+
+          Faraday.post(EnvConfig.WRC_WEBHOOK_URL) do |req|
```

Comment:
External API Request like that should be part of a job, to not slow down the server if the request is slow.

Reply:
Ah great point thank you!